File: Codechef/Beginner/FLOW011.c
#include <stdio.h>
int main()
{
    int t, sal;
    scanf("%d", &t);
    while(t--)
    {
        scanf("%d", &sal);
        if (sal < 1500)
            printf("%g\n", (double)(sal << 1));
        else
            printf("%g\n", sal + 500 + 0.98*sal);
    }
    return 0;
}

File: Codechef/Beginner/FLOW011.py
t = input()
while t:
    sal = input()
    (hra, da) = (((0.1 * sal), (0.9 * sal)) if (sal < 1500)
                 else ((500), (0.98 * sal)))
    gr_sal = sal + hra + da
    print gr_sal
    t -= 1
